### PR TITLE
ommit api call

### PR DESCRIPTION
**Pull request by vv01f**
do not try placing an order when funds are insufficient

trying to improve in small steps… not sure how to explore that node module yet for other commands

### CODE REVIEW

**Comment by dennisreimann** [comment]
Thanks for the contribution. I think this would be enough/better:

```js
if (parseFloat(fiatBalance) < parseFloat(amount)) {
  console.log('❌  Insufficient funds')
  return
}

// Place order
```

**Comment by vv01f** [comment]
true, I just have another change locally which ignores the amount but simply orders with the fiatBalance instead as it's refilled by a recurring wire anyway. just not sure how that would fit in for upstream.

**Comment by dennisreimann** [comment]
Kraken might throw an error in this case, as there are minimum amounts that need to be ordered. SO if you've got just 5 EUR left, the order won't be accepted anyways.

File: commands/stack.js [hunk 1]
```diff
@@ -48,4 +53,5 @@ module.exports = async (kraken, validate, { getEnv, getEnvOpt }) => {
 
   console.log('💸  Order:', order)
   if (txid) console.log('📎  Transaction ID:', txid.join(', '))
+
```
Comment (dennisreimann) [inline commands/stack.js:56]:
```suggestion
```

**Comment by vv01f** [comment]
@dennisreimann would you see a point in implementing commands like "show me deposit information" and a warning when deposit information changes (just in case) or "unstacking" (selling for fiat, I think for e.g. crypto accepting vendors)?

**Comment by dennisreimann** [comment]
I would not want to overcomplicate this. The script has the purpose of automating the stacking process and I haven't received any requests beyond that. 

However, if you have plans to extend it beyond this scope, feel free to fork the project.

**Pull request merged**